### PR TITLE
Added an I3S example that uses the Arcgis basemap.

### PR DESCRIPTION
**Pull request by sanyavanya**
Also fixed html title for the Mapbox-based example.

### CODE REVIEW

File: examples/website/i3s/examples.js [hunk 1]
```diff
@@ -36,7 +36,7 @@ export const EXAMPLES = {
   'New York': {
     name: 'New York',
     url:
-      'https://tiles.arcgis.com/tiles/z2tnIkrLQ2BRzr6P/arcgis/rest/services/New_York_Mesh/SceneServer/layers/0?token=epNIXupS0DwQBgeJkldNLQF_vJ7t4fKjUvas6_1mluYo11ej7BIJnOM5mPK0AJpCK1AwhnYC0Kcn7ntDlyV5_fgO9kpfhlOrnP2jCRU0UP3pRBUMB8XBeWQkvGgmcRtyU61Vx-0ioTcj8ZJsjdjFEbc24TbsUhXtVNc1BLOsvGAZjfMA_PuaFm18Cl1oOgM10C7ieRg2PJNO0-JeUHIKIgasNODDOTD0_LPqsInL_GQt_NRmnGwBN8mqElWltdy-',
+      'https://tiles.arcgis.com/tiles/P3ePLMYs2RVChkJx/arcgis/rest/services/Buildings_NewYork_17/SceneServer/layers/0',
```
Comment (ibgreen) [inline examples/website/i3s/examples.js:39]: This was broken?

File: examples/website/i3s/index.html [hunk 2]
```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <meta charset='UTF-8' />
-  <title>3D Tiles Example</title>
```
Comment (ibgreen) [inline examples/website/i3s/index.html:5]: Nice!

File: examples/website/i3s-arcgis/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "@deck.gl/arcgis": "^8.2.8",
+    "@deck.gl/core": "^8.1.4",
+    "@deck.gl/geo-layers": "^8.1.4",
+    "@deck.gl/layers": "^8.1.4",
+    "@deck.gl/mesh-layers": "^8.1.4",
+    "@loaders.gl/core": "2.2.0",
+    "@loaders.gl/i3s": "2.2.0"
+  },
+  "devDependencies": {
+    "@babel/preset-react": "^7.0.0",
```
Comment (ibgreen) [inline examples/website/i3s-arcgis/package.json:21]: No longer uses react?

**Review by ibgreen — APPROVED** [review]

File: examples/website/i3s/examples.js (hunk 1 above)
Comment (sanyavanya) [reply examples/website/i3s/examples.js:39]: Yes, it didn't work neither locally nor on the website.

**Pull request merged**